x-pack/test/security_solution_endpoint_api_int/headers.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const HEADERS = {
```

Comment:
can you `Object.freeze()` this so that it is not mutated accidentally (or convert it to a function that returns the object

Reply:
thank you for always reminding me to use `Object.freeze()` - now I made a note to not forget it always :)
00f1bc1be53df303ca642e2fe8236ac85c78dfd5